src/main/scala/cromwell/util/google/GoogleScopes.scala

```diff
@@ -1,12 +1,17 @@
 package cromwell.util.google
 
+import com.google.api.services.genomics.GenomicsScopes
 import com.google.api.services.storage.StorageScopes
+import scala.collection.JavaConverters._
 
 object GoogleScopes {
-  val Scopes = Vector(
-    StorageScopes.DEVSTORAGE_FULL_CONTROL,
-    StorageScopes.DEVSTORAGE_READ_WRITE,
-    "https://www.googleapis.com/auth/genomics",
-    "https://www.googleapis.com/auth/compute"
-  )
+  val Scopes = {
+    val scopes = Vector(
+      StorageScopes.DEVSTORAGE_FULL_CONTROL,
+      StorageScopes.DEVSTORAGE_READ_WRITE,
+      "https://www.googleapis.com/auth/genomics",
+      "https://www.googleapis.com/auth/compute"
+    ) ++ GenomicsScopes.all().asScala
```

Comment:
https://developers.google.com/resources/api-libraries/documentation/genomics/v1beta2/java/latest/com/google/api/services/genomics/GenomicsScopes.html

It is supposed to be a convenience wrapper that allows you to not list the other scopes, though DEVSTORAGE_FULL_CONTROL is not a part of it.